conda_build/cli/main_build.py

```diff
@@ -158,8 +158,10 @@ def parse_args(args):
 
 def output_action(metadata, config):
     silence_loggers(show_warnings_and_errors=False)
-    print(bldpkg_path(metadata, config))
-    sys.exit(0)
+    if metadata.skip():
+        print("skipped")
```

Comment:
This is confusing to me.  What is the point of printing just "skipped"?  You have no information about what was skipped.